e2e/app.e2e-spec.ts

```diff
@@ -47,6 +47,43 @@ describe('ng2-date-picker App', () => {
     });
   });
 
+  describe('dpDayPicker reactive directive', () => {
```

Comment:
Another question, in a reactive form the validations set within the controller (in the FormGroup). How the custom validations, MI  and max will be set? I never did it before.